bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -421,4 +436,10 @@
 		<state pattern="%.2f %unit%" readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="apEnable" advanced="false">
+		<item-type>Switch</item-type>
+		<label>Enabled</label>
+		<description>Is the access point enabled</description>
```

Comment:
Perhaps this could be changed to be consistent with `poeEnable`, as otherwise it seems more like a question (without a question mark):
```suggestion
		<description>If the access point is enabled</description>
```